src/main/java/sirius/biz/codelists/LookupTables.java

```diff
@@ -63,8 +63,8 @@ private LookupTable makeTable(String name) {
             return new CodeListLookupTable(extension, extension.get(CONFIG_KEY_CODE_LIST).asString(name));
         }
 
-        // Directly access the given IDB table..
-        LookupTable result = new IDBLookupTable(extension, jupiter.getDefault().idb().table(table));
+        // Directly access the given IDB table...
+        LookupTable result = determineIDBLookupTable(extension, name, table);
```

Comment:
nope?

Reply:
s.o.

Auch möglich wäre hier statt die Table mit `result = new IDBLookupTable(...)` neu zu laden, die über `result = fetchTable(...)` abzurufen. Aber da die Tables nicht zwingend in der richtigen Reihenfolge geladen werden, geht das nicht so einfach. Deshalb das Konstrukt in `determine/loadIDBLookupTable`.

Reply:
--> Anpassung der Ladereihenfolge nötig. Daher erst für nächste Version

Reply:
Ladereihenfolge angepasst